datamodel/extra_scripts/sqlserver/207_brk_views.sql

```diff
@@ -1471,5 +1471,7 @@ beschikbare kolommen:
 @level0type = N'Schema', @level0name = N'dbo',
 @level1type = N'View', @level1name = N'vb_kad_onrrnd_zk_archief';
 
+GO
+
 CREATE INDEX mb_kad_onrr_z_ar_overgeg_idx  ON mb_kad_onrrnd_zk_archief (overgegaan_in);
```

Comment:
deze index kan niet, geen "mb_..." views in sqlserver (sqlserver kent geen materialized views)